### ISSUE TITLE
aws 更新了ACL权限导致S3存储桶创建失败

### ISSUE THREAD

**Issue by lvtujingji** [awslabs/data-transfer-hub#111]
**Describe the bug**
当我使用2.3 cloudformation 创建 传输hub时 我遇到了这个错误
我认为是S3 4月的最新版本 里禁用了存储桶的所有权 
但我不知道改怎么去修改这个cloudformation模板
![image](https://user-images.githubusercontent.com/105871077/232373210-44139d93-98f8-45ce-a435-155ca2398f7a.png)

**Comment by lvtujingji** [comment]
这是aws S3 4月份的最新更新 ![image](https://user-images.githubusercontent.com/105871077/232373727-83fff660-f300-4b01-908d-cb39fd887c86.png)

**Comment by JoeShi** [comment]
Are you deploying the solution in us-east-2 region? The s3 ACL update has been rollout to us-east-2 (Ohio) region already, and we are working on a patch now. We are planning to release a new version in the next couple of hours.

**Comment by YikaiHu** [comment]
Hi @lvtujingji , before our patching version release, you can change the template manually.

1. Search for `PortalWebS3LoggingBucket0955B8B2`, remove `"AccessControl": "LogDeliveryWrite",` in line 6334
<img width="1710" alt="image" src="https://user-images.githubusercontent.com/34271744/232395531-cb5d027e-c595-4329-af3e-9ea07ae3207f.png">

2. Search for `PortalWebS3LoggingBucketPolicyFA081916`, add the following policy, near line 6408, please notice the format of Json.
```json
{
       "Action": "s3:PutObject",
       "Condition": {
              "ArnLike": {
                     "aws:SourceArn": {
                            "Fn::GetAtt": [
                                   "PortalWebS3Bucket93628CD2",
                                   "Arn"
                            ]
                     }
              },
              "StringEquals": {
                     "aws:SourceAccount": {
                            "Ref": "AWS::AccountId"
                     }
              }
       },
       "Effect": "Allow",
       "Principal": {
              "Service": "logging.s3.amazonaws.com"
       },
       "Resource": {
              "Fn::Join": [
                     "",
                     [
                            {
                                   "Fn::GetAtt": [
                                          "PortalWebS3LoggingBucket0955B8B2",
                                          "Arn"
                                   ]
                            },
                            "/*"
                     ]
              ]
       }
}
```

<img width="1735" alt="image" src="https://user-images.githubusercontent.com/34271744/232395806-ce088c0d-2f65-4e1c-9aa2-9551096c52dd.png">

3. Serach for PortalWebCloudfrontLoggingBucket6561051E, add the following object under `Properties`,

```json
    "OwnershipControls": {
     "Rules": [
      {
       "ObjectOwnership": "ObjectWriter"
      }
     ]
    },
```
<img width="1619" alt="image" src="https://user-images.githubusercontent.com/34271744/232397699-a50ac148-30c4-491a-98fc-79ba87441772.png">

**Comment by lvtujingji** [comment]
我在使用这套传输工具传输的时候发现，数据好像被传输了俩次 我不太理解这是我哪里配置有问题吗 从弗吉尼亚到中国北京的S3？ 还是我理解错了 
<img width="1783" alt="71151da31c2aa311cbe0bef23abdfe4" src="https://user-images.githubusercontent.com/105871077/232419282-b8c99192-1fa5-4b21-9e86-f12e1f1cb774.png">
<img width="1776" alt="699e845acc74168271cc3db9e09ba16" src="https://user-images.githubusercontent.com/105871077/232419336-bac1d9df-953a-4561-b7a2-7c52b2846c2a.png">
![53d8f7a92057f88f29a8c4bd963a11d](https://user-images.githubusercontent.com/105871077/232421584-27a71b27-60e8-43a3-9421-756c9f330961.jpg)

**Comment by YikaiHu** [comment]
请检查一下任务配置中的是否需要数据对比，建议您在该场景下选择需要数据对比。
![image](https://user-images.githubusercontent.com/34271744/232423780-2a758f8f-eda6-473b-9ff6-f3c28024dce7.png)

**Comment by lvtujingji** [comment]
感谢您的回复，那我是否可以关闭这个传输任务后，重新开启一个传输任务并开启数据对比，因为我的目标桶内未存在数据 所以我之前没有开启这个选项

**Comment by YikaiHu** [comment]
> 感谢您的回复，那我是否可以关闭这个传输任务后，重新开启一个传输任务并开启数据对比，因为我的目标桶内未存在数据 所以我之前没有开启这个选项

可以

**Comment by YikaiHu** [comment]
Hi @lvtujingji,
we have fixed this bug in v2.3.1.
You can use our latest template link to deploy the solution: 

https://solutions-reference.s3.amazonaws.com/data-transfer-hub/latest/DataTransferHub-cognito.template
